spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -301,11 +306,12 @@ describe('document-capture/components/acuant-capture', () => {
       );
 
       initialize({
-        start: sinon.stub().callsFake((_callbacks, onError) => {
+        start: sinon.stub().callsFake((callbacks) => {
+          const { onFailure } = callbacks;
           setTimeout(() => {
             const code = 'sequence-break-code';
             document.cookie = `AcuantCameraHasFailed=${code}`;
-            onError('iOS 15 sequence break', code);
+            onFailure('iOS 15 sequence break', code);
```

Comment:
Would it make sense to have two versions of these specs, one for each value of the SDK feature flag, to confirm the code still works for both values?